README.md

```diff
@@ -0,0 +1,167 @@
+ODBC FDW for PostgreSQL 9.5+
+============================
+
+This PostgreSQL extension implements a Foreign Data Wrapper (FDW) for
+remote databases using Open Database Connectivity [ODBC](http://msdn.microsoft.com/en-us/library/ms714562(v=VS.85).aspx).
+
+This was originally developed by Zheng Yang <zhengyang4k@gmail.com> in 2011,
+with contributions by Gunnar "Nick" Bluth <nick@pro-open.de> from 2014
+and further developed by CARTO <dataservices@carto.com> since 2016.
+
+Building and Installing
+-----------------------
+
+The extension can be built and installed with:
+
+```sh
+PATH=$(pg_config --bindir):$PATH make
+sudo PATH=$(pg_config --bindir):$PATH make install
+```
+
+Usage
+-----
+
+The `OPTION` clause of the `CREATE SERVER`, `CREATE FOREIGN TABLE`
+and  `IMPORT FOREIGN SCHEMA` commands is used to define both
+the ODBC attributes to define a connection to an ODBC data source
+and some additional parameters to specify the table or query that
+will be accessed as a foreign table.
+
+The following options to define ODBC attributes should be defined in
+the server definition (`CREATE SERVER`).
+
+option   | description
+-------- | -----------
+`dsn`    | The Database Source Name for the foreign database system you're connecting to.
+`driver` | The name of the ODBC driver to use (needed if no dsn is used)
+
+These additional ODBC connection options are supported and can be defined
+either in the server or foreign table definition (or in an IMPORT FOREIGN SCHEMA statement):
+
+option     | description
+---------- | -----------
+`host`     | The name of the server which provides the database.
```

Comment:
I'd simply say "the hostname or ip of the database server", or am I wrong maybe?

does it support a plain ip?